Source: Thomas Anderson
Upstream: submitted pull request https://github.com/i3/i3status/pull/24
Reason: Manpages go into /usr/share

diff -aur i3status-2.8-old/Makefile i3status-2.8/Makefile
--- i3status-2.8-old/Makefile	2014-01-05 11:34:07.000000000 +0000
+++ i3status-2.8/Makefile	2015-04-08 00:14:08.497623815 +0000
@@ -1,6 +1,9 @@
 ifndef PREFIX
   PREFIX=/usr
 endif
+ifndef MANPREFIX
+  MANPREFIX=$(PREFIX)
+endif
 ifndef SYSCONFDIR
   ifeq ($(PREFIX),/usr)
     SYSCONFDIR=/etc
@@ -93,12 +96,12 @@
 install:
 	install -m 755 -d $(DESTDIR)$(PREFIX)/bin
 	install -m 755 -d $(DESTDIR)$(SYSCONFDIR)
-	install -m 755 -d $(DESTDIR)$(PREFIX)/share/man/man1
+	install -m 755 -d $(DESTDIR)$(MANPREFIX)/share/man/man1
 	install -m 755 i3status $(DESTDIR)$(PREFIX)/bin/i3status
 	# Allow network configuration for getting the link speed
 	(which setcap && setcap cap_net_admin=ep $(DESTDIR)$(PREFIX)/bin/i3status) || true
 	install -m 644 i3status.conf $(DESTDIR)$(SYSCONFDIR)/i3status.conf
-	install -m 644 man/i3status.1 $(DESTDIR)$(PREFIX)/share/man/man1
+	install -m 644 man/i3status.1 $(DESTDIR)$(MANPREFIX)/share/man/man1
 
 release:
 	[ -f i3status-${VERSION} ] || rm -rf i3status-${VERSION}
